openvpn-wire-protocol.xml

```diff
@@ -117,50 +117,53 @@
 
     <section title="The OpenVPN Wire Protocol">
       <t>
-        Since OpenVPN can work both in a traditional server-client setup
-        as well as a peer-to-peer setup, this document tries to avoid the
-        concept of server and clients.  It will refer to these as
-        either the local or remote sites. In a peer-to-peer setup only
-        a single tunnel can be established, while in a server-client setup
-        several clients can connect to a single server at the same time.
-
-        If the terms server and client are used these are almost always
-        synonymous with the peer's role as either TLS server or TLS client.
+        OpenVPN can work both in a traditional client-server topology
+        and in a peer-to-peer topology.  In a peer-to-peer setup only
+        a single point-to-point tunnel can be established, while in a
+        client-server setup multiple clients can connect to a single
+        server in a star configuration.
+      </t>
+      <t>
+        When the terms client and server are used in the context of a
+        peer-to-peer topology, this reflects the peer&apos;s role as
+        either TLS server or TLS client.
       </t>
 
       <section title="TCP and UDP transport modes">
         <t>
           OpenVPN is capable of using both UDP and TCP for transporting
           SSL/TLS traffic.  The SSL/TLS protocol is strictly written for
-          TCP but OpenVPN makes that possible through encapsulating the
-          SSL/TLS packets and adding a reliability layer to avoid issues
-          when packets get resent.
+          TCP but OpenVPN makes UDP possible by encapsulating the
+          SSL/TLS concepts in the payload and adding a reliability
+          layer in the payload to avoid issues when packets get
+          resent.
         </t>
       </section>
-      <section title="Basic OpenVPN packet format">
+
+      <section title="OpenVPN payload format">
```

Comment:
I would avoid using "payload" here since we use the same term later for the VPN playload inside the OpenVPN packets.

Reply:
@flichtenheld I see what you mean.  As in the example of layout:
`-opcode- || -session_id- || -packet_id- || auth_tag || * payload *`
and in the chart heading in the "Overview of OPCODEs" section.

My suggestion here is that the use of "payload" in both of those examples is unconventional.  My usage follows the convention that the payload is what comes after the headers.  And the body of the section you identify in the comment _**only**_ addresses the payload, not the full packet.  To reconcile that with the unconventional usage, I would suggest calling the additional data that follows opcodes and other stuff at the front of the payload "more data", "control data",  "optional data", "extra data", or "vpn traffic" depending on the specific context.  I would be happy to add those changes to the PR.

Reply:
A couple references for conventional vs unconventional use of payload:
https://en.wikipedia.org/wiki/Network_packet
https://networkengineering.stackexchange.com/questions/35016/whats-the-difference-between-frame-packet-and-payload

and there are more by googling packet payload.